### PR TITLE
Import of multiclient account(s) 

### PR DESCRIPTION
**Pull request by gbayasgalan**

### CODE REVIEW

File: stereum-control-center-cli.sh [hunk 1]
```diff
@@ -10,6 +10,38 @@ dialog_title="Stereum Control Center"
...
+    --inputbox "Please enter the password of the accounts:" 9 60 "" \
+    3>&1 1>&2 2>&3)
+
+  choice_validator_mnemonic=$(dialog --backtitle "$dialog_backtitle" \
+    --title "$dialog_title" \
+    --inputbox "Please enter the mnemonic of the validator_keys:" 9 60 "" \
+    3>&1 1>&2 2>&3)
+
+  ansible-playbook \
+    -e validator_keys_path="$choice_launchpad_wallet_path" \
```
Comment (stefa2k) [inline stereum-control-center-cli.sh]: var `validator_keys_path` is never used by the playbook/role

File: stereum-control-center-cli.sh [hunk 2]
```diff
@@ -10,6 +10,38 @@ dialog_title="Stereum Control Center"
 stereum_config_file_path=/etc/stereum/ethereum2.yaml
 
 function dialog_import_wallet() {
+  if [ "$setup" == "multiclient" ]; then
+  choice_launchpad_wallet_path=$(dialog --backtitle "$dialog_backtitle" \
```
Comment (stefa2k) [inline stereum-control-center-cli.sh]: var validator_keys_path is never used by the playbook/role, so this dialog is not necessary imo

File: stereum-control-center-cli.sh [hunk 3]
```diff
@@ -10,6 +10,38 @@ dialog_title="Stereum Control Center"
...
+    --title "$dialog_title" \
+    --inputbox "Please enter the directory of the validator_keys\n(e. g. /tmp/validator_keys):" 9 60 "" \
+    3>&1 1>&2 2>&3)
+
+  choice_validator_number=$(dialog --backtitle "$dialog_backtitle" \
+    --title "$dialog_title" \
+    --inputbox "Please enter the number of the validator_keys:" 9 60 "" \
+    3>&1 1>&2 2>&3)
+
+  choice_account_password=$(dialog --backtitle "$dialog_backtitle" \
```
Comment (stefa2k) [inline stereum-control-center-cli.sh]: See comment in review of https://github.com/stereum-dev/ethereum2-ansible/pull/37 about using the same auto generated passphrase for both: wallet and account

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: stereum-control-center-cli.sh (hunk 1 above)
Comment (gbayasgalan) [reply stereum-control-center-cli.sh]: var `validator_keys_path` is removed

File: stereum-control-center-cli.sh (hunk 2 above)
Comment (gbayasgalan) [reply stereum-control-center-cli.sh]: yes, you rigth, this dialog is removed

File: stereum-control-center-cli.sh (hunk 3 above)
Comment (gbayasgalan) [reply stereum-control-center-cli.sh]: var `validator_password` also `password dialog` is removed. Create-account & wallet going to use `auto-generated` passphrase

**Review by stefa2k — APPROVED** [review]
lgtm

**Pull request merged**